components/todo/TodoItem.vue

```diff
@@ -29,23 +55,43 @@ export default {
   setup() {
     return {
       format,
+      formatDistance,
+      bn,
     }
   },
   data() {
     return {}
   },
+
+  computed: {
+    isTodoCompleted() {
+      return this.todo.completedAt
```

Comment:
while using `null` or `date` works for this computed value, being not of the same type might cause problems if we use this value. We could add a property to todo items called `isTodoComplete` which will specifically keep track of completeness, instead of using the `completeAt` date.

Reply:
resolved